docs/stable/index.yaml

```diff
@@ -1,3 +0,0 @@
-apiVersion: v1
```

Comment:
```
apiVersion: v1
entries: {}
generated: 1970-01-01T00:00:00.000000000-00:00
```
for both docs/stable/index.yaml and docs/staging/index.yaml

This ensures that if either are unable to be generated (no charts, for instance) an empty index file will still exist.